### DISCUSSION TITLE
Login vs Client ID/Secret

### DISCUSSION THREAD

**Discussion by TheGreatBeginner777** [KRTirtho/spotube#346] [Q&A] [answered]
Hi.
I'm following [Getting Started With Spotube](https://spotube.netlify.app/blog/getting-started-with-spotube). At the moment I managed to do all the steps up to 

> Copy the Client ID & paste it in the Spotube’s Text Field. Then tap/click on the SHOW CLIENT SECRET button to reveal the Client Secret. Finally, copy the Client Secret & paste it in the Spotube’s corresponding Text Field

Where should I enter Client ID and Client Secret? 
My way of login. From bottom bar -> Settings -> Account -> Login with your Spotify account -> (click on it) -> (pop up) CONNECT WITH SPOTIFY -> (click on it) -> enter email and password used for login into developer.spotify.com. 
I'm unable to figure out what is my status with this type of login? Where should I see my Client ID and Client Secret.

**Comment by TheGreatBeginner777** [comment, marked as answer]
Spotube version 2.6.0
Android version 13.

**Reply by KRTirtho** [reply]
Version greater than 2.5.0 doesn't require ClientID and Client Secret anymore...
You can just L:gin using your email + password in Mobile

**Comment by TheGreatBeginner777** [comment]
Thanks for this info! After login is it possible to check my account info? I don't know if that is even necessary for the purpose of using this app.

**Reply by KRTirtho** [reply]
You can't see your account info in the app. This is why https://accounts.spotify.com is available

**Reply by TheGreatBeginner777** [reply]
Thanks a lot on your feedback.
